cmd/mesh/manifest-diff.go

```diff
@@ -33,11 +33,23 @@ const YAMLSuffix = ".yaml"
 type manifestDiffArgs struct {
 	// compareDir indicates comparison between directory.
 	compareDir bool
+	// selection specifies the resources to compare, it's comma-separated of resource indicators
+	// in the form of "<kind>:<namespace>:<name>,<kind>:<namespace>:<name>".
```

Comment:
suggest changing names to selectResources and ignoreResources, as selection is a bit confusing.
also, i propose the comment to read "selectResources constrains the list of resources to compare to only the ones in this list, ignoring all others. The format of each list item is <kind>:<namespace>:<name> and the items are comma separated. The * character represents wildcard selection. e.g. 
Deployment:istio-system:* - compare all deployments in istio-system namespace
Service:*:istio-pilot - compare Services called "istio-pilot" in all namespaces.

Reply:
done.